### DISCUSSION TITLE
Para quem esta com problemas ao enviar email (Gmail)

### DISCUSSION THREAD

**Discussion by jsfachini** [RamonSilva20/mapos#1848] [General]
Para quem esta com "problema" no Gmail, ao enviar e-mails pelo Mapos ou outro app, da pra resolver desta forma.

Você tem que ir em >Gerenciar conta do google>Segurança>Como fazer login no google>Senha de app.
nessa parte da senha de app, você escolhe selecionar app >E-mail > selecionar dispositivo> Windows e clica em GERAR
vai criar uma senha especifica para uma aplicação como um MS Outlook ou Mapos, essa senha não da para acessar a conta do google, ela é exclusiva para uso em algum APP.

Obs: Lembrando que a conta precisa estar com a verificação em duas etapas ATIVADO

![EMAILGOOGLE](https://user-images.githubusercontent.com/72340074/175028525-a499c0f0-4b1f-4742-88c6-6429bece1258.png)

**Comment by matheusneves2023** [comment]
Poderia colocar um passo a passo de como configurar na hostgator?
